### PR TITLE
Feat, Style: Improve Import Icon Activity 

### PR DESCRIPTION
**Pull request by nethical6**
Todo:

~Replace pngs with vectors~
~Make vectors configurable~

### CODE REVIEW

**Comment by ilyassesalama** [comment]
Finally someone did it, great job!

**Comment by Yamenher** [comment]
C'mon, I've been waiting for weeks for this pr to get merged, can you @nethical6 just finish it, that would be sooo freaking awesome 😃🔥

File: app/src/main/res/layout/manage_locallibraries.xml [hunk 1]
```diff
@@ -1,140 +1,144 @@
-<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:fitsSystemWindows="true">
-
-    <!-- please set lift on scroll to false and elevation to 0 -->
-    <!-- when you implement search -->
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (remaker17) [inline app/src/main/res/layout/manage_locallibraries.xml:1]: Change PR title because of this changes.

**Review by remaker17 — CHANGES_REQUESTED** [review]

**Comment by remaker17** [comment]
Can you also add some screenshots of your work?

**Comment by nethical6** [comment]
Known issues:
1. when setting background resource of a layout as a vector, the preview screen will fail to display it. however the background resource would be visible in the final compiled project.

**Pull request merged**